packages/terra-date-picker/src/react-datepicker/date_utils.js

```diff
@@ -383,7 +383,7 @@ export function getLocalizedDateForScreenReader (date, props) {
 
   if (date && date.isValid()) {
     const localizedDate = localizeDate(date, locale);
-    localizedDateLabel = localizedDate.format('LL');
+    localizedDateLabel = localizedDate.format('LLLL');
```

Comment:
Does `LLLL` include the time? https://github.com/moment/moment/blob/develop/locale/en-au.js#L30. Would it conflict with the time-input when used in date-time-picker?

Reply:
Yes, the `LLLL` format would have conflicted with the date-time-picker. I have updated the format to be `dddd, LL`.